plugins/ida/bap_ida_service.ml

```diff
@@ -0,0 +1,154 @@
+open Core_kernel.Std
```

Comment:
Initially, yes, but after that, so many changes were made, that overall, git does not recognize it as a movement of file anymore (basically, it's heuristic is not working anymore). See here: https://git.wiki.kernel.org/index.php/GitFaq#Why_does_Git_not_.22track.22_renames.3F